src/components/VideoPlayerContainer.vue

```diff
@@ -14,7 +14,7 @@
     </div>
     <!-- TAP TO UNMUTE OVERLAY -->
     <div
-      v-if="(isLive && !this.viewer.muted && this.viewer.autoplay)"
+      v-if="autoPlayMuted && isLive && !this.viewer.muted"
```

Comment:
@dubeyShivank @keon-jukes I went with a simpler route to check if the query param is muted. It appears this.viewer is an object of the options that were initially passed to the player (basically the query params) and they don't change. The controls.muted does change but we can just check viewer options here and that will tell us if the query param was set or not.